mobilecoind-dev-faucet/README.md

```diff
@@ -55,27 +86,104 @@ Other options are:
 - `--listen-host` - hostname for webserver, default `127.0.0.1`
 - `--listen-port` - port for webserver, default `9090`
 - `--mobilecoind-uri` - URI for connecting to mobilecoind gRPC, default `insecure-mobilecoind://127.0.0.1:4444/`
-- `--target-queue-depth` - The number of pre-split transactions the faucet attempts to maintain in its queue. Default is 20.
+- `--target-queue-depth` - The number of pre-split transactions the faucet attempts to maintain in its queue. Default is 500.
 - `--worker-poll-period-ms` - A lower bound on how often the worker thread wakes up to check in with `mobilecoind`. Default is `100` milliseconds.
 
 ### Usage with cURL
 
-Here is some example usage:
+It is relatively straightforward to test the faucet locally using the `tools/local_network.py` script. First simply follow those instructions to start a local network from your shell.
+
+(depending on if you run in docker or not your ledger base may be different)
+
+```
+$ cargo build --release
+$ ./tools/local_network/bootstrap.sh
+$ export LEDGER_BASE=/tmp/mobilenode/target/sample_data/ledger
+$ export MC_LOG=info
+$ ./tools/local_network/local_network.py --network-type dense5 --skip-build &
+```
+
+Then, start a faucet and set it to also work in the background:
+```
+$ ./target/docker/release/mobilecoind-dev-faucet --keyfile "$LEDGER_BASE/../keys/account_keys_0.json" &
```

Comment:
the actual steps I had used `target/docker/release` because I was doing this in the container. I just changed the `#` to `$` for the docu. But I'm happy to change this anyways to be consistent with that.